en/docs/guides/user-mgt/lock-account.md

```diff
@@ -8,6 +8,67 @@ Account locking and account disabling are security features in WSO2 Identity Ser
 
 ---
 
+## Lock user accounts using SCIM
+
+1. Open the `<IS-HOME>/repository/conf/scim2-schema-extension.config` file and add the following configuration. 
+
+   ```
+   {
+      "attributeURI":"urn:ietf:params:scim:schemas:extension:enterprise:2.0:User:accountLock",
+      "attributeName":"accountLock",
+      "dataType":"boolean",
+      "multiValued":"false",
+      "description":"Lock user account.",
+      "required":"false",
+      "caseExact":"false",
+      "mutability":"readwrite",
+      "returned":"default",
+      "uniqueness":"none",
+      "subAttributes":"null",
+      "canonicalValues":[],
+      "referenceTypes":[]
+   }
+   ```
+
+2. Then add the `accountLock` attribute as a sub-attribute of User.
+
+   ```
+   "subAttributes":"verifyEmail askPassword accountLock employeeNumber costCenter organization division department manager"
+   ```
+
+3. Save the file and restart the server. 
+
+Before locking/unlocking users using SCIM, you need to do the following. 
+
+1. [Add claim mapping](insert-link)
```

Comment:
we can probably include a fragment for this

Reply:
I don't think this is used anywhere else so a fragment would be an overhead IMO

Reply:
adding claim mapping is a common thing right?

Reply:
Yes, claim mapping should be a fragment but it's better if it's a separate page and linked off to. Otherwise these pages might become too text heavy. WDYT?